test/packages/parallel/gcp/data_stream/compute/fields/ecs.yml

```diff
@@ -1,5 +1,3 @@
-- external: ecs
-  name: cloud
```

Comment:
Not sure about how we want to support this, I see two options:
* As it is in current `main` branch, importing the definition of the group, that is used only for documentation. We would need to use `ecs_nested.yml` instead of `ecs_flat.yml` for this in this PR.
  This may be error prone, as it may look like if someone can use the `cloud` field.
* Importing the whole object, making easier to import groups of ECS fields.
* Don't import anything, and require explicit imports, as is done by now in this PR.

Reply:
> * As it is in current `main` branch, importing the definition of the group, that is used only for documentation. We would need to use `ecs_nested.yml` instead of `ecs_flat.yml` for this in this PR.

I am going to do this, otherwise this is going to be breaking for many packages.